guide/14-deep-learning/object_detection_and_tracking_on_videos.ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<p>Can you add a note directing readers to https://github.com/Esri/arcgis-python-api/blob/master/samples/04_gis_analysts_data_scientists/automate_road_surface_investigation_using_deep_learning.ipynb for the full workflow?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>